etc/picongpu/cori-nersc/a100_picongpu.profile.example

```diff
@@ -21,6 +21,7 @@ export RESERVATION=""
 
 # General modules #############################################################
 #
+module purge
 module load dgx
 module swap PrgEnv-intel PrgEnv-gnu
 module load cuda openmpi
```

Comment:
There is more to it, but have to hop in a meeting.

Trying
```suggestion
module purge
module load dgx
module load gcc
module load cuda
module load openmpi
```
but I did something else to get it working last time..

Reply:
Should I wait, for the integration of this suggestion, before I merge this PR?